openshift/templates/eagle-api.dc.json

```diff
@@ -232,6 +232,10 @@
                               "name": "${CHES_SECRET_NAME}"
                           }
                       }
+                  },
+                  {
```

Comment:
Was already added below and was just missing from here.

Reply:
👍